### ISSUE TITLE
Users dropdown in top right corner doesn't work on mobile

### ISSUE THREAD

**Issue by peter-drew** [MelbourneHighSchoolRobotics/RCJA_Registration_System#258]
Could either remove because have hamburger menu or fix

**Comment by Dalordish** [comment]
@peter-drew I'm assuming this is a mobile browser issue because it works fine for me on desktop emulating mobile screens.

Honestly, upon having another look at the top bar, having so much content there on mobile is problematic regardless.

![image](https://user-images.githubusercontent.com/5959205/76159625-9f028880-6176-11ea-8707-287f78afd255.png)

I guess the real question is:

As an admin on mobile, what am I likely to be doing
As a multi-school mentor, what am I likely to be doing
As a single-school mentor, what am I likely to be doing,
and: As a mentor AND admin, what am I likely to be doing

At this point I'm inclined to say that mobile users most of the time really won't be caring about their own details or administering their school, but if you are a multi-school mentor, you probably care a great deal about which school you're interacting as, and so that should be prioritised as an always visible menu item. Similarly, as an admin, you probably care most about being able to access admin, so that should also be prioritised

**Comment by peter-drew** [comment]
I agree that the school selector and admin buttons should be prioritied. We can dump the user dropdown menu. Will need to test if the school selector works on mobile, although this is a low priority issue, the site isn't mobile optimised and entering a lot of data on mobile would be horrible anyway.

**Comment by peter-drew** [comment]
See also #89
